docs/version0.18.md

```diff
@@ -37,6 +37,10 @@ The following new features and notable changes since v 0.17.0 are included in th
 
 The [`jextract` tool](tool_jextract.md) is now available on macOS&reg; platforms (as well as AIX&reg; and Linux&trade;) for _all_ current versions of OpenJDK: 8, 11, and 13.
 
+### `-Xmso` 1 MB minimum value on z/OS 64-bit
+
+On z/OS 64-bit, [`-Xmso`](xmso.md) has a 1 MB minimum value, to match the minimum stack space provided by the OS. Setting a value smaller than 1 MB is ignored.
+
```

Comment:
- Need a link to this section from the top of the topic - you can use the jextract one as an example (currently there's an "Add more changes here" link that needs removing, so you could just change that). MkDocs generates html IDs for headings by making the heading text lowercase and adding dashes between words. So the ID for your section is -xmso-1-mb-minimum-value-on-zos-64-bit.
- Expand "OS" in "provided by the OS" to "operating system"
- For the last sentence, I suggest "If you set a value that is smaller than 1 MB, the value is ignored."

Reply:
Done

Reply:
I left in the "Add more changes here" because I think it helps avoid merge conflicts.

Reply:
lgtm, thanks